roles/mesos-agent/tasks/main.yml

```diff
@@ -15,13 +18,13 @@
     - { file: '/etc/mesos-slave/hostname', content: "{{mesos_hostname}}" }
     - { file: '/etc/mesos-slave/containerizers', content: 'docker,mesos' }
     - { file: '/etc/mesos-slave/isolation', content: 'cgroups/cpu,cgroups/mem' }
-    - { file: '/etc/mesos-slave/executor_registration_timeout', content: '5mins' }
+    - { file: '/etc/mesos-slave/executor_registration_timeout', content: '2mins' }
```

Comment:
What is the reason for lowering this? For large docker images I think it is better to have this higher to increase the change of the download succeeding before the Mesos task fails